Makefile

```diff
@@ -27,7 +27,7 @@ CRI:=docker # nerdctl
 OPENAPI_CODEGEN_TAG=v6.4.0
 OPENAPI_CODEGEN_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_CODEGEN_TAG}
 DOCKER_OPENAPI=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${OPENAPI_CODEGEN_IMAGE}
-OPENAPI_URL= https://app.swaggerhub.com/apiproxy/registry/equinix-api/fabric/4.11
+OPENAPI_URL="https://app.swaggerhub.com/apiproxy/registry/equinix-api/fabric/4.11"
```

Comment:
Where can I learn more about this registry `https://app.swaggerhub.com/apiproxy/registry/equinix-api`?  Do you know if it's widely used by Equinix API teams?  If this is, generally speaking, the source of truth for API specs across Equinix, it might be worth establishing some patterns around it for use in `equinix-sdk-go`.  For example: it might be easier to update the spec version if it were in a separate `SPEC_VERSION` variable that is referenced in the `OPENAPI_URL` variable.

Reply:
From what I can tell it's just a url format that will return each page in data format only without a UI page being returned. It's just an alternate URL to "https://api.swaggerhub.com/apis/equinix-api/fabric/4.11/swagger.yaml" that gives the same result, but in JSON. It's identical to "https://api.swaggerhub.com/apis/equinix-api/fabric/4.11/swagger.json".

AFAIK, there isn't a standardized registry for all of equinix. It could be that all equinix API Specs are under `https://api.swaggerhub.com/apis/equinix-api/` but I couldn't say it with certainty at this point.